src/database-controller/build/build-pre.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
```

Comment:
FY20 CELA update: copyright headers no longer need to have "All rights reserved" appended.
https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html

Reply:
Done.